### PR TITLE
Include options from superclass commands (such as -o)

### PR DESCRIPTION
**Pull request by jkeiser**

### CODE REVIEW

**Comment by eeyun** [comment]
I need this as well. :+1:

**Comment by robbkidd** [comment]
Diving in on understand it!

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+source "https://rubygems.org"
+gemspec
+gem 'chef'
```
Comment (robbkidd) [inline Gemfile]: The `chef` gem is already declared as a dependency in the gemspec.

Comment (robbkidd) [inline Gemfile]: A super old version of the chef gem, but declared nonetheless.

**Comment by robbkidd** [comment]
Testing manually. (We really need to add some tests.)

@jkeiser I'm going to omit the `gem 'chef'` from Gemfile and bump the plugin gem's version.

**Comment by robbkidd** [comment]
:+1:

**Pull request merged**